pyat/at/physics/radiation.py

```diff
@@ -23,8 +22,7 @@
                   ('emitXYZ', numpy.float64, (3,))]
 
 
-def ohmi_envelope(ring, refpts=None, orbit=None, keep_lattice=False,
-                  energy=None):
+def ohmi_envelope(ring, refpts=None, orbit=None, keep_lattice=False):
```

Comment:
We should update the docstring to remove energy and say that a Lattice object is required.

Reply:
Done!